recipes/hdf5-lz4/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.2" %}
+{% set git_sha = "8fa813aa369c15127c551cfdeebe4cbc090b4687" %} # no tags in repo :-(
+
+package:
+    name: hdf5-lz4
+    version: {{ version }}
+
+source:
+    url: https://github.com/dectris/HDF5Plugin/archive/{{ git_sha }}.tar.gz
+    fn: hdf5-lz4-{{ version }}.tar.gz
+    sha256: 232cb6b80bccbfbc782d4d10b467cffbba316306fb97ebd249b9fd0fc2490539
+
+build:
+    number: 4
+    skip: True  # [not linux]
+
+requirements:
+    build:
+        - hdf5 1.8.15*
+    run:
+        - hdf5 1.8.15*
+test:
+    requires:
+        - h5py
+    commands:
+        - HDF5_PLUGIN_PATH=$CONDA_ENV_PATH/lib python -c "import h5py"
```

Comment:
I think I found a better way to test that the plugin is found, just need to test it.